src/GeneralStateTestsFiller/stMemoryTest/callDataCopyOffsetFiller.json

```diff
@@ -0,0 +1,70 @@
+{
+    "callDataCopyOffset" : {
+        "env" : {
+            "currentCoinbase" : "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x20000",
+            "currentGasLimit" : "1000000",
+            "currentNumber" : "1",
+            "currentTimestamp" : "1000",
+            "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "expect" : [
+            {
+                "indexes" : {
+                    "data" : -1,
+                    "gas" : -1,
+                    "value" : -1
+                },
+                "network" : ["ALL"],
+                "result" : {
+                    "eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee" : {
+                        "storage" : {
+                            "0x" : "0xffffffffffffffffffffffffffffffff"
+                        }
+                    }
+                }
+            }
+        ],
+        "pre" : {
+						"eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee" : {
+                "balance" : "1000000000000000000",
+								"//codeDescription": "/* fill memory up */ /* copy call data to memory with offset > call data length */  /* only part of memory should be zeroed out */",
+								"code": "{ (MSTORE 0x00 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff) (CALLDATACOPY 0x00 0xffff  0x10) (SSTORE 0x00 (MLOAD 0x00)) }",
+                "nonce" : "1",
+								"storage": {
+								}
```

Comment:
And around here, too?

Sometimes I read test fillers on GitHub, and currently, I need to scroll to the right.